setup-env.sh

```diff
@@ -362,6 +362,7 @@ fi
     LICENSE_FILE="$(dirname ${LICENSE_FILE})/$(basename ${LICENSE_FILE})"
   fi
   export LICENSE_FILE_PATH="- ${LICENSE_FILE}:/etc/wandisco/fusion/server/license.key"
+  export LICENSE_FILE_PATH_ONE_UI="- ${LICENSE_FILE}:/etc/wandisco/fusion/server/license/license.key"
```

Comment:
Correct me if i'm wrong, but does Fusion Server should be aware of this new license path (etc/wandisco/fusion/server/license/license.key)? if it does, how Fusion Server knows?